pyfr/solvers/euler/kernels/entropyfilter.mako

```diff
@@ -102,6 +102,32 @@
     // Filter if out of bounds
     if (dmin < ${d_min} || pmin < ${p_min} || emin < entmin - ${e_tol})
     {
+        % if linearize:
+        // Compute mean quantities
+        fpdtype_t uavg[${nvars}], davg, pavg, eavg;
+        % for vidx in range(nvars):
+        uavg[${vidx}] = ${' + '.join(f'{jx}*u[{j}][{vidx}]'
+                                     for j, jx in enumerate(meanwts) if jx != 0)};
+        % endfor
+
+        ${pyfr.expand('compute_entropy', 'uavg', 'davg', 'pavg', 'eavg')};
+
+        // Apply density, pressure, and entropy limiting sequentially
+        fpdtype_t alpha;
+        % for (fvar, bound) in [('d', f'{d_min}'), ('p', f'{p_min}'), ('e', f'entmin - {e_tol}')]:
+        if (${f'{fvar}min < {bound}'}) 
```

Comment:
No need for an f string here; can use native Mako ${}.

Reply:
Do you mean something similar to what I just pushed or different?

Reply:
`if (${fvar}min < ${bound})`